scripts/Engineering/gui/engineering_diffraction/tabs/fitting/plotting/plot_view.py

```diff
@@ -10,16 +10,11 @@
 from qtpy.QtWidgets import QDockWidget, QMainWindow, QMenu
 from mantidqt.utils.qt import load_ui
 from matplotlib.figure import Figure
-from matplotlib.backends.qt_compat import is_pyqt5
+from matplotlib.backends.backend_qt5agg import FigureCanvas
```

Comment:
A while back we introduced a [shim](https://github.com/mantidproject/mantid/blob/5a7a30803c08b9281498af64899e004a3f8259d3/qt/python/mantidqt/MPLwidgets.py) to try and consolidate imports from the matplotlib Qt backends. Can we import the classes from this `MPLWidgets.py` module instead so that then we can reduce the number of places we have Qt version checks?

Similarly for the other uses of `backend_qt5agg.py` that have been updated?